credentials/apps/edx_credentials_extensions/edly_credentials_app/api/v1/views/program_certificate_configuration.py

```diff
@@ -0,0 +1,46 @@
+"""
+Views for Edly Program Certificate Configuration API.
+"""
+from rest_framework import status, viewsets
+from rest_framework.permissions import IsAuthenticated
+from rest_framework.response import Response
+
+from credentials.apps.credentials.models import ProgramCertificate
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.permissions import CanAccessCurrentSite
+from credentials.apps.edx_credentials_extensions.edly_credentials_app.api.serializers import (
+    ProgramCertificateConfigurationSerializer
+)
+
+
+class ProgramCertificateConfigurationViewSet(viewsets.ModelViewSet):
```

Comment:
I feel we have a lot of code which can be omitted. Why cant we have single viewset to manage Program certificate data and signatories data and that viewset should have a single serializer ProgramCertificate serializer. ProgramCertificate serializer can have a RelatedField to signatories model with one.
https://www.django-rest-framework.org/api-guide/relations/

Reply:
@ziafazal You are right, related fields relate to a single field of a model. We wanted to send all necessary fields for signatories from WordPress for creation & updation. We used the Site as a relationship because we wanted to just create relation we don't want to create a site via this API. 
We already used a single viewset to handle requests.
```
site = serializers.SlugRelatedField(slug_field='domain', queryset=Site.objects.all())
```
Here's an example JSON data that we will be sending from WordPress for the creation of a program certificate configuration.
```
{
    "site": "edx.devstack.lms:18150",
    "is_active": true,
    "signatories": [
        {
            "name": "Jhon Doe",
            "title": "Director Certificates",
            "image": "data:image/png;base64,VBORw0KGgoAAAANSUhEUgAAAAEAAAABCAQAAAC1HAwCAAAAC0lEQVR42mNk+A8AAQUBAScY42YAAAAASUVORK5CYII="
        }
    ],
    "program_uuid": "242681e5-b746-4204-8601-f0512a23b6fb",
    "use_org_name": true,
    "include_hours_of_effort": true,
    "language": "en"
}
```